src/bootctl/bootctl-status.c

```diff
@@ -478,7 +478,7 @@ int verb_status(int argc, char *argv[], void *userdata) {
 
                         sd_id128_t loader_partition_uuid = SD_ID128_NULL;
                         (void) efi_loader_get_device_part_uuid(&loader_partition_uuid);
-                        print_yes_no_line(/* first= */ false, !sd_id128_is_null(loader_partition_uuid), "Boot loader set partition information");
+                        print_yes_no_line(/* first= */ false, !sd_id128_is_null(loader_partition_uuid), "Boot loader sets partition information");
```

Comment:
hmm, hmm, hmm. 

so ths doesn't make it better really, it makes the whole thing wrong in another way...

this is simply not a feature here, it shouldn't show up under "Features"... it should be its own category because it doesn't indicate what is theoretically supported but what was actually passed.

Reply:
It's still a feature ;) We know that feature is implemented because the boot loader passed the feature to us. I don't think it's useful to split this section in two just to distinguish how exactly we learned that a feature is implemented.

Reply:
well, the field is not set in all cases. for example it doesn't apply to net booted stuff and suchlik.

hence, it's not a feature, it's an optional field that might or might not be passed down to use, depending how the system is booted.

Reply:
You are right. I looked at the output more closely, and amended the commit to drop both lines ;---]. PTAL!